src/backend_helper.c

```diff
@@ -16,6 +18,16 @@ static unsigned int HttpLocalTimeout = 5;
 
 Mappings *map;
 
+char *get_printer_name_for_cups_dest(const cups_dest_t *dest)
+{
+    assert(dest);
```

Comment:
Please do not use `assert()` in production code, let the function return `NULL` instead, so that the caller can handle the error.

Reply:
Done. (Though every single caller would previously have crashed on a null dest, as it was using `dest->name` directly.)